frameworks/JavaScript/express/app.js

```diff
@@ -36,7 +36,7 @@ var FortuneSchema = new mongoose.Schema({
   MFortune = conn.model('Fortune', FortuneSchema);
 
 var sequelize = new Sequelize('hello_world', 'benchmarkdbuser', 'benchmarkdbpass', {
-  host: 'localhost',
+  host: 'TFB-database',
```

Comment:
It was unintentional but ok nonetheless. We can actually begin changing all of the statements like this to TFB-database and get rid of the messy sed statements in the setup files.